core/controllers/question_editor.py

```diff
@@ -56,8 +58,9 @@ def post(self, skill_id):
         question_services.add_question(self.user_id, question)
         # TODO(vinitamurthi): Replace DEFAULT_SKILL_DIFFICULTY
         # with a value passed from the frontend.
-        question_services.create_new_question_skill_link(
-            question.id, skill_id, constants.DEFAULT_SKILL_DIFFICULTY)
+        for skill_id in skill_ids:
+            question_services.create_new_question_skill_link(
```

Comment:
Ditto -- look into implementing some kind of create_multi/put_multi method if you're going to save multiple datastore entities at once.

Reply:
(That said, please ignore this comment and similar comments below if the method doesn't actually touch the datastore.)

Reply:
This is being done in my PR #6929